### PR TITLE
Ankita_Asgnmnt_29

### PR DESCRIPTION
**Pull request by AnkitaDOct2020**
image

### CODE REVIEW

**Review by harshvegada — CHANGES_REQUESTED** [review]
Hi @AnkitaDOct2020 
Please take look of this code & update code, acknowledge me if you get idea of this below code
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/931/files

File: src/ankita/assignment/array/SecondMaxNumber.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+import java.util.Arrays;
+
+public class SecondMaxNumber {
+
+	void findSecondMaxNumber(int[] arr) {
+		int maxFirst = 0;
+		int maxSecond = 0;
+
+		if (arr[0] > arr[1]) {
```
Comment (harshvegada) [inline src/ankita/assignment/array/SecondMaxNumber.java:15]: Why you applied if condition here?

File: src/ankita/assignment/array/SecondMaxNumber.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+		} else {
+			maxFirst = arr[1];
+			maxSecond = arr[0];
+		}
+		for (int i = 2; i < arr.length; i++) {
+			if (arr[i] > maxFirst) {
+				maxSecond = maxFirst;
+				maxFirst = arr[i];
+
+			} else if (arr[i] > maxSecond) {
```
Comment (harshvegada) [inline src/ankita/assignment/array/SecondMaxNumber.java:27]: else if (numArr[i] > secondLargest && numArr[i] != largest) {
	secondLargest = numArr[i];
}

**Review by harshvegada — APPROVED** [review]
Code give correct output but you need to put on condition in else if,
Take of this next time

**Comment by AnkitaDOct2020** [comment]
> Code give correct output but you need to put on condition in else if,
> Take of this next time

Yes Harsh, will take care of this from next time.

**Pull request merged**